### ISSUE TITLE
The sharpen algorithms it's the same with “Registax  Wavelets” ?

### ISSUE THREAD

**Issue by nosqldb** [Rolf-Hempel/PlanetarySystemStacker#42]
Hi Rolf, in your CN post you mentioned:
"It turned out that Registax and PSS use the same sharpening filters"
I review code here ,it seems to use unsharp mask actually ,it's the same with “Registax  Wavelets” ？

By the way, I also tried the bilateralfilter ,but it seems no imporvement than gauss,(imporvement  is not easy)

**Comment by Rolf-Hempel** [comment]
Hi nosqldb,

When I had implemented and published my first version of PSS postprocessing, Cor Berrevoets, the Author of Registax, in a private communication revealed to me that his software uses the same hierarchy of filters (i.e. Gaussian unsharp masking). At that point my scheme followed the "linked layers" approach of Registax. Later I decoupled the layers, so that now PSS and Registax 6 use the same overall scheme.

Using the bilateral filters is not as straightforward as using Gaussian filters. With the appropriate settings they suppress overshootings at sharp edges very effectively, though. I now use a mix of Gaussian and bilateral filters on a regular basis with great success.

All the best, 
 Rolf
